src/PowerShellEditorServices/Services/TextDocument/Handlers/CompletionHandler.cs

```diff
@@ -97,34 +80,32 @@ public static bool CanResolve(CompletionItem value)
         // Handler for "completionItem/resolve". In VSCode this is fired when a completion item is highlighted in the completion list.
         public async Task<CompletionItem> Handle(CompletionItem request, CancellationToken cancellationToken)
         {
-            // We currently only support this request for anything that returns a CommandInfo: functions, cmdlets, aliases.
-            if (request.Kind != CompletionItemKind.Function)
-            {
-                return request;
-            }
-
-            // No details means the module hasn't been imported yet and Intellisense shouldn't import the module to get this info.
-            if (request.Detail is null)
+            // We currently only support this request for anything that returns a CommandInfo:
+            // functions, cmdlets, aliases. No detail means the module hasn't been imported yet and
+            // IntelliSense shouldn't import the module to get this info.
+            if (request.Kind != CompletionItemKind.Function || request.Detail is null)
```

Comment:
```suggestion
            if (request.Kind is not CompletionItemKind.Function || request.Detail is null)
```

nit: `is` can be used whenever the RHS is a constant